### PR TITLE
Add "fruits"

### PR DESCRIPTION
**Pull request by muukii0803**

### CODE REVIEW

**Comment by jurre** [comment]
Looks great @muukii0803, could you just move it over to the `color_schemes` array?

**Comment by muukii0803** [comment]
@jurre Sorry, I've just fixed it.

**Comment by jurre** [comment]
Great, thanks!

File: packages.json [hunk 1]
```diff
@@ -786,6 +786,12 @@
         "screenshot": "https://raw.githubusercontent.com/etsushisa/Flat-Theme-Xcode/master/Screen%20Shot%20Light.png"
       },
       {
+        "name" : "fruits",
+        "url" : "https://github.com/muukii0803/xcode-fruits",
```
Comment (jurre) [inline packages.json]: Sorry, but this should be:

```json
"url": "https://raw.githubusercontent.com/muukii0803/xcode-fruits/master/fruits.dvtcolortheme",
```

**Comment by muukii0803** [comment]
@jurre Oh,, Sorry..

**Comment by jurre** [comment]
Awesome, thanks!

**Pull request merged**